docs/CSBR.md

```diff
@@ -14,14 +14,69 @@ copyright: |
+
+`{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1) code-signing-requirements(4) code signing(1)} (2.23.140.1.4.1)`.
+
+The following Certificate Policy Identifier is reserved for use by CAs as a required means of asserting compliance with these Requirements for EV Code Signing Certificates follows:
+
+`{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1) code-signing-requirements(3)}(2.23.140.1.3)`.
+
+The following Certificate Policy Identifier is reserved for use by CAs as a required means of asserting compliance with these Requirements for Timestamp Certificates:
+
+`{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1) code-signing-requirements(4) timestamping(2)} (2.23.140.1.4.2)`.
+
+### 1.2.1 Revisions
+
+| **Ver.** | **Ballot** | **Description** | **Effective** |
+|-|-|-----|--|
+| 1.2 | CSC-1 | Adopt Baseline Requirements version 1.2 | 13 Aug 2019 |
+| 2.0 | CSC-2 | Adopt combined EV and BR Code Signing Document | 2 Sept 2020 |
+| 2.1 | CSC-4 | Move deadline for transition to RSA-3072 and SHA-2 timestamp tokens | 7 Nov 2020 |
+| 2.2 | CSC-7 | Update to merge EV and non-EV clauses | 8 March 2021 |
+| 2.3 | CSC-8 | Update to Revocation response mechanisms. key protection for EV certificates, and clean-up of 11.2.1 & Appendix B | 2 May 2021 |
+| 2.4 | CSC-9 | Spring 2021 Clean-up and Clarification | 8 September 2021 |
+| 2.5 | CSC-10 | WebTrust CSBR v2.0 Audit Criteria | 12 September 2021 |
+| 2.6 | CSC-11 | Update to log data retention requirements | 3 November 2021 |
+
+### 1.2.2 Relevant Dates
+
+| **Compliance** | **Section(s)** | **Summary Description (See Full Text for Details)** |
+|--|--|----------|
+| 2021-06-01 | 6.1.5 | CAs SHALL support minimum RSA-3072 for Code Signing Certificates, Root Certificates and Subordinate CA Certificates. CAs SHALL NOT support SHA-1 digest algorithm for Code Signing Certificates.|
+| 2021-06-01 | 5.3 | After 2021-06-01, the CA shall meet the requirements of EV Guidelines Section 14.1 for Non-EV and EV Code Signing Certificates.|
+| 2021-06-01 | 6.2.7.4 | For EV Code Signing Certificates, Signing Services shall protect private keys in a FIPS 140-2 level 2 (or equivalent) crypto module. After 2021-06-01, the same protection requirements SHALL apply to Non EV Code Signing Certificates.|
+| 2021-11-01 | 3.2.2.1 (5) | The method used to verify the identity of the Certificate Requester SHALL be per section 3.2.3.|
+| 2022-03-31 | 7.1.6.3 | Subordinate CA Certificates issued for Subordinate CA that issues Timestamp Certificates and is an Affiliate of the Issuing CA must include the reserved identifier specified in Section 7.1.6.1.|
+| 2022-04-30 | 7.1.3.2.1 | CAs SHALL NOT support SHA-1 digest algorithm for Timestamp tokens.|
+
 ## 1.3  PKI participants
 
 ### 1.3.1  Certification authorities
 
 ### 1.3.2  Registration authorities

```

Comment:
The existing section 14.2 refers to "Registration Authorities and Subcontractors". I wonder if it would be a violation to include the "subcontractors" in the title of section 1.3.2. After all, this section also talks about "signing services" which are far from "Registration Authorities".